go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/jarcoal/httpmock v1.1.0
 	github.com/sirupsen/logrus v1.8.1
 	github.com/stretchr/testify v1.7.0
+	go.uber.org/zap v1.21.0
```

Comment:
Seems this is only used for a no-op logger (unless I missed something). Not sure it's worth the dependency, were you planning on keeping this or is it just temporary?

Reply:
good point! going to implement a no-op logger instead of using this